pkg/certificate/certificate.go

```diff
@@ -91,3 +96,47 @@ func NewFromPEM(pemCert pem.Certificate, pemKey pem.PrivateKey) (*Certificate, e
 		Expiration:   x509Cert.NotAfter,
 	}, nil
 }
+
+// CreateValidCertAndKey creates a non-expiring PEM certificate and private key
+func CreateValidCertAndKey(notBefore, notAfter time.Time) (pem.Certificate, pem.PrivateKey, error) {
```

Comment:
Is this method only used by tests? Maybe it would make sense in a test helper file or something like that?